### PR TITLE
Chapter 5 translated to Russian 

### PR DESCRIPTION
**Pull request by Lightmourne**
The translation of chapter 5 has been completed.

Hello, @MKhalusova! I think I found a typo in the evaluation.mdx (line 203) I think the word "target" should be changed to "prediction".
Please let me know if I am wrong.

### CODE REVIEW

File: chapters/en/chapter5/evaluation.mdx [hunk 1]
```diff
@@ -200,7 +200,7 @@ normalized_prediction
 ```
 
 Great! We can see that the text has been fully lower-cased and all punctuation removed. Let's now define the reference
-transcription and then compute the normalised WER between the reference and target:
+transcription and then compute the normalised WER between the reference and prediction:
```
Comment (sanchit-gandhi) [inline chapters/en/chapter5/evaluation.mdx:203]: Good catch!

**Review by sanchit-gandhi — COMMENTED** [review]
Thank you for your contribution @Lightmourne! Let's wait for @MKhalusova to take a look here before merging :)

**Review by MKhalusova — CHANGES_REQUESTED** [review]
Thank you for your contribution! The translation looks good to me, but there are some checks failing. Please run `make style` to fix formatting issues.

**Comment by Lightmourne** [comment]
Thank you @MKhalusova !
File formatting was checked :white_check_mark:

**Pull request merged**